src/ai/backend/client/cli/vfolder.py

```diff
@@ -101,10 +105,12 @@ def create(name, host, group, host_path, usage_mode, permission):
         try:
             if host_path:
                 result = session.VFolder.create(name=name, unmanaged_path=host, group=group,
-                                                usage_mode=usage_mode, permission=permission)
+                                                usage_mode=usage_mode, permission=permission,
+                                                quota=quota)
```

Comment:
Let's break the lines here in a more black-like style.
```suggestion
                result = session.VFolder.create(
                    name=name,
                    unmanaged_path=host,
                    group=group,
                    usage_mode=usage_mode,
                    permission=permission,
                    quota=quota,
                )
```